src/pyodide/python-entrypoint-helper.js

```diff
@@ -219,20 +230,23 @@ function getMainModule() {
     // TODO: investigate whether it is possible to run part of loadPyodide in top level scope
     // When we do it in top level scope we seem to get a broken file system.
     const pyodide = await loadPyodide();
-    return await setupPackages(pyodide);
+    const mainModule = await setupPackages(pyodide);
+    const relaxed_call = pyodide.pyimport("relaxed_call").relaxed_call;
+    result = { mainModule, relaxed_call };
+    return result;
   })();
   return mainModulePromise;
 }
 
 export default {
-  async fetch(request, env) {
-    const mainModule = await getMainModule();
-    return await mainModule.fetch(request);
+  async fetch(request, env, ctx) {
+    const { relaxed_call, mainModule } = await getMainModule();
```

Comment:
```suggestion
    const { relaxed_call, mainModule } = await getPyodide();
```

feel free to choose a better name than my suggestion :)